source/bin.ts

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
-'use strict'
 
-const validate = require('./')
+import validate from './'
```

Comment:
use `'./index.js'` instead of `'./'` - we should include filenames and `.js` extensions to make lookups quicker (although ts is .ts, we do .js as that is what is run at compile-time)